7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>I'm confused, I thought this is supposed to be between 0 and 1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Me, too...</p><p> </p><p>But sklearn uses a difference definition. It is the <code>R2_adjusted</code> term </p><p>from the website: "This is not a symmetric function.</p><p>Unlike most other scores, </p><p>score may be negative (it need not actually be the square of a quantity R)."</p><p> </p><p><a href="https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html" target="_blank">https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html</a></p><p> </p><p>Blew my mind too...</p>